code/game/objects/machinery/doors/poddoor.dm

```diff
@@ -232,6 +232,7 @@
 	icon = 'icons/obj/doors/mainship/blastdoors_shutters.dmi'
 	name = "Timed Emergency Shutters"
 	use_power = FALSE
+	resistance_flags = DROPSHIP_IMMUNE|RESIST_ALL
```

Comment:
All this does is make normal timed podlocks not breakable at all. The subtype you are looking for (aka fob shutters) has the correct var to edit at l. 246.
Although having dropship_immune alone on normal timed podlocks wouldn't be a bad thing per se.